### PR TITLE
Add ZHHTo4B and WHHTo4B madgraph LO,NLO cards and gen fragment (Run 3)

### PR DESCRIPTION
**Pull request by chayanit**
Dear GEN experts, 

I have prepared MG cards for ZHH to 4b and WHH to 4b non resonant processes for Run-3 analyses. These processes make use of VBF HH's model to modify coupling parameters (kappa_V, kappa_VV and kappa_lambda) at LO. We also need NLO WHH and ZHH (including loop-induced ggZHH diagrams) at SM point to reweigh signals to NLO and NNLO, respectively, as done in Run-2 (HIG-22-006). 

Best,
Chayanit for VHH team

### CODE REVIEW

**Review by menglu21 — COMMENTED** [review]
just one minor comment. @Saptaparna @mlizzo @Dominic-Stafford please have a look

File: genfragments/ThirteenPointSixTeV/Higgs/HH/VHH_HToBB_HToBB_M-125_TuneCP5_13p6TeV-madgraph-pythia8.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
```
[the remark this answers is not in the document]
Comment (chayanit) [reply genfragments/ThirteenPointSixTeV/Higgs/HH/VHH_HToBB_HToBB_M-125_TuneCP5_13p6TeV-madgraph-pythia8.py]: Hi @menglu21 , I have updated as you suggested.

**Comment by menglu21** [comment]
thanks for the update. @Saptaparna @mlizzo @Dominic-Stafford please have a check

**Comment by mlizzo** [comment]
Hi, cards and the fragment look fine to me

**Pull request merged**